src/odemis/acq/stream/_projection.py

```diff
@@ -1660,7 +1660,11 @@ def _updateImage(self):
         try:
             data = self._computeSpec()
             if data is not None:
-                self.image.value = self._project2RGB(data, self.stream.tint.value)
+                tint = self.stream.tint.value
+                if tint == TINT_FIT_TO_RGB:
+                    # Not supported for this view => fallback to greyscale
```

Comment:
Would it make sense to log this info, so that it doesn't just change silently without any feedback for the user?

Reply:
Well, that's mostly due to a limitation/simplification in the GUI: the user can only select 1 type of colormap/tint per stream, although the stream has multiple projections (in this case, there are 4 possible projections, typically all shown simultaneously on the screen). So the GUI allows to select "Fit to RGB" as for one projection it's meaningful. For all the other ones, it doesn't make sense, and we just use the "standard" tint instead. I could add a log message, but no one is going to look at it. I could make it a actual pop up in the GUI and it's just going to annoying everyone... so I think doing nothing is fine.